charts/ingress-nginx/templates/check.yaml

```diff
@@ -0,0 +1 @@
+{{ include  "isControllerTagValid" . | required "controller container image tag should be 0.27.0 or higher" }}
```

Comment:
@ChiefAlexander this does not work but contains the idea of tag validation. Any idea how this can be done?